android-app/src/main/java/org/example/app/units/NewsListUnitsFactory.kt

```diff
@@ -6,10 +6,11 @@ package org.example.app.units
 
 import dev.icerock.moko.resources.desc.StringDesc
 import dev.icerock.moko.units.TableUnitItem
+import javax.inject.Inject
 import org.example.app.TileNews
 import org.example.library.SharedFactory
 
-class NewsListUnitsFactory : SharedFactory.NewsUnitsFactory {
+class NewsListUnitsFactory @Inject constructor() : SharedFactory.NewsUnitsFactory {
```

Comment:
can we not use `@Inject` consturctor? :)

Reply:
No, we can't. We need constructor's injection for `UnitFactoryModule`, because it uses interface's implementation to provide.